src/main/groovy/be/cytomine/software/communication/SSH.groovy

```diff
@@ -47,10 +49,19 @@ class SSH implements Communication {
             Properties properties = new Properties()
             properties.put("StrictHostKeyChecking", "no")
 
+            printf(Main.configFile.cytomine.software.softDev)
+            if(Main.configFile.cytomine.software.softDev)
+            {
+                if(user=="cytomine")
+                {
+                    host=Main.configFile.cytomine.software.slurmip
+                    port=22
+                }
+            }
```

Comment:
That is by far too specific. What if an other processing server make a ssh connection using "cytomine" as user ?

Instead of this, the bootstrap which get the dynamic IP of slurm container should put it into your /etc/hosts.
If the slurm container IP is 172.17.0.12 for example, put 
172.17.0.12 slurm in you /etc/hosts

Then you can contact slurm from you machine with 
ssh cytomine@slurm

I tried it works.